README.md

```diff
@@ -449,6 +449,84 @@ Note that because this method is not available in the generic `IDataReader` inte
 TimeSpan timeSpanTime = ((SnowflakeDbDataReader)reader).GetTimeSpan(13);
 ```
 
+Executing a Batch of SQL Statements (Multi-Statement Support)
+--------------------------------------------------------------
+
+With version 2.0.18 and later of the .NET connector, you can send
+a batch of SQL statements, separated by semicolons,
+to be executed in a single request.
+
+---
+**Note**
+
+By default, Snowflake returns an error for queries issued with multiple statements to protect against SQL injection attacks. The multiple statements feature makes your system more vulnerable to SQL injections, and so it should be used carefully. You can reduce the risk by using the MULTI_STATEMENT_COUNT parameter to specify the number of statements to be executed, which makes it more difficult to inject a statement by appending to it.
+
+---
+
+You can execute multiple statements as a batch in the same way you execute queries with single statements, except that the query string contains multiple statements separated by semicolons. Note that multiple statements execute sequentially, not in parallel. The MULTI_STATEMENT_COUNT parameter specifies the exact number of statements the batch contains.
+
+For example, if you set MULTI_STATEMENT_COUNT=3, a batch statement must include precisely three statements. If you submit a batch statement with any other number of statements, the Node.js driver rejects the request. You can set MULTI_STATEMENT_COUNT=0 to allow batch queries to contain any number of statements. However, be aware that using this value reduces the protection against SQL injection attacks.
+
+You can set this parameter at the session level using the following command, or you can set the value separately each time you submit a query.
+
+```
+ALTER SESSION SET MULTI_STATEMENT_COUNT = <n>;
+```
+
+By setting the value the session level, you do not need to set it when you execute each time you execute a batch statement. The following example sets the number of statements at the session level to three and then executes three SQL statements:
```

Comment:
The example seems to set the parameter for a specific set of statements (i.e. sets a statement-level parameter), rather than setting the parameter for the session.

Also, the text here says that the number of statements is set to three, but the example shows MULTI_STATEMENT_COUNT being set to 0.

Reply:
Thanks. I forgot my .NET sample didn't have a call to set the session variable. Will work with Dicky to get an update.